Formula/grc.rb

```diff
@@ -34,8 +35,17 @@ def post_install
   end
 
   def caveats; <<-EOS.undent
-    New shell sessions will start using GRC after you add this to your profile:
-      . #{etc}/grc.bashrc
+    To use GRC automatically for new shell sessions:
+
+      Bash: add to ~/.profile:
+        . /usr/local/etc/grc.bashrc
```

Comment:
Use `#{etc}` instead of `/usr/local/etc`. Not all users have Homebrew under `/usr/local`.